src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search--bars--container">
```

Comment:
Try removing the container all-together, just have two forms with margin-top: 20px and margin-left: 8px.

After that run `npm run test:only` and adjust values for line-height and others styles to fit the reference
```suggestion
```